### PR TITLE
Beautify generated java code

### PR DESCRIPTION
**Pull request by kormide**
@jbedard

### CODE REVIEW

File: recipe-generator/src/main/java/ca/derekcormier/recipe/generator/JavaGenerator.java [hunk 1]
```diff
@@ -29,4 +34,20 @@ public JavaGenerator(Cookbook cookbook) {
             new JavaGetterFilter()
         );
     }
+
+    @Override
+    protected String renderTemplate(String templatePath, Map<String, Object> data) {
+        String generatedJava = super.renderTemplate(templatePath, data);
+
+        Formatter formatter = new Formatter(JavaFormatterOptions.builder().style(JavaFormatterOptions.Style.AOSP).build());
```
Comment (jbedard) [inline recipe-generator/src/main/java/ca/derekcormier/recipe/generator/JavaGenerator.java]: You prefer AOSP over the standard google one? idk the differene...

File: recipe-generator/src/main/java/ca/derekcormier/recipe/generator/JavaGenerator.java [hunk 2]
```diff
@@ -29,4 +34,20 @@ public JavaGenerator(Cookbook cookbook) {
...
+        String prettyJava;
+        try {
+            prettyJava = formatter.formatSource(generatedJava);
```
Comment (jbedard) [inline recipe-generator/src/main/java/ca/derekcormier/recipe/generator/JavaGenerator.java]: Could just `return ...` here... :man_shrugging:

File: recipe-generator/src/main/java/ca/derekcormier/recipe/generator/JavaGenerator.java (hunk 1 above)
Comment (kormide) [reply recipe-generator/src/main/java/ca/derekcormier/recipe/generator/JavaGenerator.java]: I just saw that AOPS had 4 space indents while google had 2 space indents.

**Comment by kormide** [comment]
Updated

**Review by jbedard — APPROVED** [review]

**Pull request merged**